autogen/outputs.tf.tmpl

```diff
@@ -25,3 +25,8 @@ output "instance_group" {
   description = "Instance-group url of managed instance group"
   value       = google_compute_region_instance_group_manager.{{ module_name }}.instance_group
 }
+
+output "instance_group_manager" {
+  description = "An instance of google_compute_region_instance_group_manager of the instance group."
+  value       = google_compute_region_instance_group_manager.mig
```

Comment:
This will need to differ based on. the module name. (See line 26).

Reply:
addressed in 2752517d2de110c12e6c2bd04f02b984d0282ad0